tasks/system_probe.py

```diff
@@ -198,8 +198,9 @@ def kitchen_prepare(ctx, windows=is_windows):
     # test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg/network/netlink/testsuite
     # test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg/ebpf/testsuite
     # test/kitchen/site-cookbooks/dd-system-probe-check/files/default/tests/pkg/ebpf/bytecode/testsuite
+    gopath = check_output("go env GOPATH", shell=True).decode('utf-8').strip("\n")
```

Comment:
With go modules, you are no longer required to have your code in the GOPATH.

Reply:
good point, I move the logic with regex as we just want a short path (ex: pkg/network) at the end
And it would be less painful to support on all OS